electron/anticheat/titles_map.ts

```diff
@@ -0,0 +1,27 @@
+// this hash contains a mapping of titles as displayed in Epic to titles as
+// displayed in areweanticheatyet.com https://github.com/Starz0r/AreWeAntiCheatYet/blob/master/games.json
+//
+// comparison is case insensitive, so no need to add cases like 'VALORANT': 'Valorant'
+// if the epic title is contained, so no need to add cases like 'Fall Guts': 'Fall Guys: Ultimate Knockout'
+//
+// left is the epic title, right is the areweanticheatyet title
+const TITLES_MAP: { [key: string]: string } = {
+  'Battlefield™ 2042': 'Battlefield 2042',
+  'Back 4 Blood: Standard Edition': 'Back 4 Blood',
```

Comment:
> if the epic title is contained, so no need to add cases like 'Fall Guts': 'Fall Guys: Ultimate Knockout'

Why not the opposite too?

Reply:
hmmm honestly? I don't remember now (I did this over 2 weeks about on a late night and manually checked the +200 games in the games.json file one by one in the epic store xD)

I think I thought about that and for some reason I decided to keep it simple and only in this direction

Maybe it works just fine doing the check the other way around, but I know that for example I had to add this entry `'The Division': "Tom Clancy's The Division"` to prevent a false match. I suspect that some of the game names are also false matches if you compare them the other way around (I can't confirm if tthis was the case thought, and I wouldn't go over all the games again by hand to check this claim haha)

If somebody is willing to check them all again I'm ok with that change, but it would be just to remove some entries from this mapping and maybe add some if needed, I don't know if it's worth it.